### PR TITLE
bashremainder: stop running after 15:00

### LINKED ISSUE #1: bash_remainder: after a specific time the script is closed
As example after the 15:00 I don't need anymore the remainder so the script can closed

### PR DESCRIPTION
**Pull request by vitorgalvao**
Closes https://github.com/Mte90/My-Scripts/issues/1.

Also made some style fixes.

### CODE REVIEW

File: misc/bashremainder.sh [hunk 1]
```diff
@@ -2,17 +2,18 @@
...
+cutoff_hour=15
+runme='firefox "http://www.ibreviary.com/m2/breviario.php?s=lodi"'
 message='Vuoi fare le lodi?'
 
-seconds=$(($minutes * 60))
+seconds="$((${minutes} * 60))"
 
 while true; do
-    $(kdialog --yesno "$message")
-    if [ "$?" = 0 ]; then
```
Comment (vitorgalvao) [inline misc/bashremainder.sh:12]: Checking for the whole command is better than `$?`.

File: misc/bashremainder.sh [hunk 2]
```diff
@@ -2,17 +2,18 @@
...
-        $($runme)
-        break
```
Comment (vitorgalvao) [inline misc/bashremainder.sh:14]: Unecessary.

File: misc/bashremainder.sh [hunk 3]
```diff
@@ -2,17 +2,18 @@
...
+    [[ "$(date +%H)" -ge 15 ]] && exit
+
+    if kdialog --yesno "${message}"; then
+        ${runme}
```
Comment (vitorgalvao) [inline misc/bashremainder.sh:15]: The surrounding `$()` is unecessary (at least on macOS).

File: misc/bashremainder.sh [hunk 4]
```diff
@@ -2,17 +2,18 @@
...
 
-seconds=$(($minutes * 60))
+seconds="$((${minutes} * 60))"
 
 while true; do
-    $(kdialog --yesno "$message")
-    if [ "$?" = 0 ]; then
-        $($runme)
-        break
+    [[ "$(date +%H)" -ge 15 ]] && exit
```
Comment (vitorgalvao) [inline misc/bashremainder.sh:12]: Check for the current hour (without minutes), and if it is greater or equal to `15`, exit the script.

**Comment by Mte90** [comment]
Thank you also for the style coding issue (I forgot to check them)

**Pull request merged**